src/sgame/sg_bot_util.cpp

```diff
@@ -1649,6 +1650,19 @@ void BotFireWeaponAI( gentity_t *self )
 	}
 }
 
+void BotChangeClass( gentity_t *self, class_t newClass, vec3_t newOrigin )
```

Comment:
maybe `inline`?

Reply:
It never makes sense to use `inline` in a non-header

Reply:
so I guess it's fine as it is?

Reply:
fine as it is